azure-pipelines.yml

```diff
@@ -141,6 +141,16 @@ jobs:
         - Linux_x64
         - Windows_NT_x64
         - Windows_NT_x86
+      ${{ if eq(variables['Build.DefinitionName'], 'coreclr-outerloop-jitstress-isas-arm') }}:
+        - Linux_arm
+        - Linux_arm64
+        - Windows_NT_arm
+        - Windows_NT_arm64
+      ${{ if eq(variables['Build.DefinitionName'], 'coreclr-outerloop-jitstress-isas-x86') }}:
+        - Linux_x64
+        - OSX_x64
```

Comment:
Linux and OSX cover the same calling conventions and general codegen, but the OSX machines have caught a number of bugs due to a number of the machines in the pool having AVX but not AVX2 support.

This has been impactful for a number of cases for the older System.Numerics.Vector code when refactoring the JIT.